frontend/src/metabase/parameters/components/ParameterValueWidget.css

```diff
@@ -37,28 +37,34 @@
   border-color: var(--color-brand);
 }
 
+:local(.parameter) input {
+  flex-basis: 142px;
```

Comment:
Do we need this?

```suggestion
```

Reply:
Tried it out locally and didn't notice any differences with and without `flex-basis`. @oleggromov could you double check? If it's not needed let's drop it

Reply:
Yeah, agreed. We either need `min-width` or explicit `flex-shrink: 0` for that to work, or remove it. I will remove.